drmemory/docs/fuzzer.dox

```diff
@@ -177,6 +177,22 @@ you would specify the following command-line option to \p drmemory:
 
```

Comment:
If you could add a simple test of this so we have a regression test that exercises the option that should finish this off.